chromium/browser_type.go

```diff
@@ -255,11 +255,16 @@ func executablePath(
 	path string,
 	env env.LookupFunc,
 	lookPath func(file string) (string, error), // os.LookPath
-) (outPath string, ok bool) {
-	if strings.TrimSpace(path) != "" {
-		return path, true
+) (string, error) {
+	// find the browser executable in the user provided path
+	if path := strings.TrimSpace(path); path != "" {
+		if _, err := lookPath(path); err == nil {
+			return path, nil
+		}
+		return "", ErrChromeNotInstalled
```

Comment:
I wonder if it would make it clearer if there was a different error returned here, such as "chrome nor chromium not found at given path"

Reply:
I was considering the same, TBH 👍 It might be not very clear for users to see Chrome/Chrome not installed while they're providing the path. These two are separate cases. See: 8227bb0.